reporthandling/helpers/v1/listing.go

```diff
@@ -1,145 +1,154 @@
 package helpers
 
 import (
+	"sync"
+
 	"github.com/kubescape/opa-utils/reporthandling/apis"
-	"github.com/kubescape/opa-utils/reporthandling/internal/slices"
+	"golang.org/x/exp/maps"
 )
 
+var allListsPool = &sync.Pool{
+	New: func() interface{} {
+		return &AllLists{}
+	},
+}
+
+// GetAllListsFromPool get the AllLists object from the pool
+func GetAllListsFromPool() *AllLists {
+	l := allListsPool.Get().(*AllLists)
```

Comment:
can we have ,ok check to verify conversion is ok?

Reply:
https://github.com/kubescape/opa-utils/pull/108